src/components/comment-input.component.js

```diff
@@ -90,6 +92,13 @@ export class CommentInput extends Component {
 
     return (
       <View style={styles.container}>
+        <MentionArea
+          trigger="@"
+          text={this.state.text}
+          updateText={text => this.setState({ text })}
```

Comment:
Actually now that I look at it we use arrow functions like this in a lot of places throughout the code. It's probably best to update this all together in a separate PR and open an issue.